tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -1,9 +1,19 @@
 local LG_funcs = {}
 
-function LG_funcs.init(dsize, csize)
+function LG_funcs.init(dsize, csize, large_val_perc, large_val_sz)
     LG_funcs.dsize = dsize
     LG_funcs.csize = csize
     LG_funcs.esize = math.ceil(dsize / csize)
+    LG_funcs.huge_value_percentage = large_val_perc
+    LG_funcs.huge_value_size = large_val_sz
+end
+
+local function huge_entry()
```

Comment:
I would like to expose this as a metric such that once the seeder finishes it will preempt how many big values it created. However, since this code is a script I don't see a "smart way". 

Maybe a seeder can create a key in dragonfly (`set big_values number_of_big_values`) which can then the poll ? 

@chakaz any ideas/thoughts?

Reply:
I thought about this and we don't really need `scan`. In fact I baked this metric in the lua script which we return -- works perfectly